### DISCUSSION TITLE
Cannot interact with content in div.drawer-content

### DISCUSSION THREAD

**Discussion by DJ-caddev** [saadeghi/daisyui#1826] [Q&A] [answered]
Hello,

I'm using the last version of DaisyUI in a vue3 + typecript 5 project.

I am trying to use the drawer, but the [first example from the documentation](https://daisyui.com/components/drawer/#drawer) does not work on my side.

I have been trying on differents browers and tried to downgrade DaisyUI to 2.27, but I still cannot click on the "open drawer" button or select any text from elements present under 'div.drawer-content'.
I also tried to copy-paste the whole block of the example into my website base, but same thing: no interaction possible.

Is there any precise step to make the drawer work or should it work out of the box?

Thank you for this amazing UI library!

EDIT : the example in the document website works well, I forgot to tell that ^^

**Comment by saadeghi** [comment]
Is your Tailwind config is correct, it should work.  
Can you share a small repo where I can see your code?

**Reply by DJ-caddev** [reply]
That is what I thought too.
I will try to make a small reproducible example then.
Thank you for the fast answer :)

**Reply by DJ-caddev** [reply, marked as answer]
I found the culprit:

By default, Vue used to ship some CSS, such as:
```
*,
*::before,
*::after {
  box-sizing: border-box;
  margin: 0;
  position: relative;
  font-weight: normal;
}
```
in src/assets/base.css.

But I think they realized it was a "bad practice", as the maintainers of create-vue decided to remove the "position: relative;" line, in the [very last version of the library](https://github.com/vuejs/create-vue/releases/tag/v3.6.4):
* Removed * { position: relative } from base.css to avoid pollution of global style

Below is a minimal project including TailwindCSS + DaisyUI above Vue3 only. Adding "position: relative" from "base.css" makes possible the reproduction of my "bug", if we can call that as such.
[vue-project.zip](https://github.com/saadeghi/daisyui/files/11488050/vue-project.zip)
